scripts/userstyles.yml

```diff
@@ -434,29 +434,29 @@ userstyles:
       past-maintainers: [*andreasgrafen]
   mdn:
     name: MDN
-    category: development
+    categories: [development]
```

Comment:
MDN is for devlopers but also is used for education, and potentially a wiki?

Reply:
I wouldn't say `education` but I agree the second category should be `wiki` or whatever.

Reply:
That makes sense 👍